File: app/src/main/java/com/gcoders/schoolinfo/schoolsfindersystem/utils/SchoolUtils.java
package com.gcoders.schoolinfo.schoolsfindersystem.utils;

public class SchoolUtils {

    public static boolean isThisInteger(String strInput) {
        try {
            Integer.parseInt(strInput);
            return true;
        } catch (NumberFormatException ex) {
            return false;
        }
    }
}

File: app/src/main/java/com/gcoders/schoolinfo/schoolsfindersystem/utils/SchoolUtils.kt
package com.gcoders.schoolinfo.schoolsfindersystem.utils

object SchoolUtils {

    fun isThisInteger(strInput: String): Boolean {
        try {
            Integer.parseInt(strInput)
            return true
        } catch (ex: NumberFormatException) {
            return false
        }

    }
}
